core/src/tools/bsmtp.cc

```diff
@@ -598,18 +559,17 @@ int main(int argc, char* argv[])
   }
 #endif
 
-  fprintf(sfp, "To: %s", argv[0]);
-  Dmsg1(10, "To: %s", argv[0]);
-  for (i = 1; i < argc; i++) {
-    fprintf(sfp, ",%s", argv[i]);
-    Dmsg1(10, ",%s", argv[i]);
-  }
+  BStringList list_of_recipients;
+  list_of_recipients << recipients;
+  fprintf(sfp, "To: %s", list_of_recipients.Join(',').c_str());
+  Dmsg1(10, "To: %s", list_of_recipients.Join(',').c_str());
+
 
   fprintf(sfp, "\r\n");
   Dmsg0(10, "\r\n");
-  if (cc_addr) {
-    fprintf(sfp, "Cc: %s\r\n", cc_addr);
-    Dmsg1(10, "Cc: %s\r\n", cc_addr);
+  if (!cc_addr.empty()) {
```

Comment:
Question: will it be complicated to handle easily and safely a list of cc, same for reply-to ?

Reply:
should be quite straight forward

Reply:
As I don't see a real use case and nobody ask for, we let cc being a single address.